### ISSUE TITLE
⛔ 🐞 Bug Report: I'm executing elementrFrontend.init(); and e-lottie are being duplicated

### ISSUE THREAD

**Issue by ofmarconi** [elementor/elementor#21390]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I'm calling a page via Fetch:

```
fetch('https://site .com/?p=1170')
  .then(response => response.text())
  .then(data => {
// Manipulate the returned data, for example:
    console.log(data);
    document.getElementById('case').innerHTML = data;

// Re-enable Elementor Pro entrance animations
    if (typeof elementorFrontend !== 'undefined') {
      elementorFrontend.init();
    }
  })
  .catch(error => console.error(error));
```

It turns out that if I don't run the snippet: elementorFrontend.init(); animations for new elements in the DOM do not start.

And adding the existing e-lottie elements are duplicated (I just identified those elements being duplicated)

Is this a Bug?
I imagine so, why should I duplicate it?

Thanks!

### Steps to reproduce

Add this javascript exchanged for the values of your site and page that contain animations

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: Ubuntu 20.04 v10.3.37
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://donestudiodigital.com
	Home URL: https://donestudiodigital.com
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: pt-BR
	Timezone: -3
	Admin Email: marconi@marks.art.br
	Debug Mode: Inactive

== Theme ==
	Name: By Marks.art.br
	Version: 1.1.4.1632168674
	Author: Marconi
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.4.2
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: pt_BR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Admin Columns Pro
		Version: 5.6.4
		Author: AdminColumns.com

	Advanced Database Cleaner PRO
		Version: 3.1.7
		Author: Younes JFR.

	All-in-One WP Migration
		Version: 7.70
		Author: ServMask

	All-in-One WP Migration Unlimited Extension
		Version: 2.48
		Author: ServMask

	Disable REST API
		Version: 1.7
		Author: Dave McHale

	Duplicate Page
		Version: 4.5.1
		Author: mndpsingh287

	Elementor
		Version: 3.11.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.1
		Author: Elementor.com

	JetBlog For Elementor
		Version: 2.3.1
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.6.9
		Author: Crocoblock

	JetEngine
		Version: 3.1.1
		Author: Crocoblock

	JetFormBuilder
		Version: 3.0.0.2
		Author: Crocoblock

	JetPopup
		Version: 2.0.0
		Author: Crocoblock

	JetTricks
		Version: 1.4.3
		Author: Crocoblock

	ManageWP - Worker
		Version: 4.9.16
		Author: GoDaddy

	Post Types Order
		Version: 1.9.9.2
		Author: Nsp Code

	Query Monitor
		Version: 3.11.1
		Author: John Blackbourn

	Retarget Marks
		Version: 2.35
		Author: Marconi

	Slider Revolution
		Version: 6.6.10
		Author: ThemePunch

	Slider Revolution Typewriter Effect
		Version: 3.0.6
		Author: ThemePunch

	Super Page Cache for Cloudflare
		Version: 4.7.3
		Author: Optimole

	WP Offload SES Lite
		Version: 1.6.0
		Author: Delicious Brains

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 1.0.0
		Author: GoDaddy


== Uso dos elementos ==
	
	container : 2
		container : 2
		jet-listing-grid : 2
	page : 1
		button : 1
		container : 1
		heading : 2
	popup : 2
		button : 1
		container : 1
		form : 1
		heading : 1
		icon : 2
		jet-inline-svg : 1
		lottie : 1
	section : 1
		icon-list : 1
		jet-inline-svg : 1
	wp-page : 5
		animated-headline : 1
		button : 2
		container : 15
		form : 1
		heading : 3
		html : 4
		image : 8
		jet-inline-svg : 3
		jet-listing-grid : 1
		lottie : 3
		slider_revolution : 1
	single-post : 1
		button : 1
		container : 29
		heading : 15
		icon : 1
		image : 12
		jet-inline-svg : 1
		jet-listing-dynamic-terms : 1
		lottie : 1
		text-editor : 10
		video : 2
	jet-popup : 1
		container : 1
		shortcode : 1
	jet-listing-items : 0
		button : 2
		container : 2
		heading : 4
		jet-inline-svg : 1


== Configurações ==
	
	cpt_support: post, page, jet-popup
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: fallback


== Recursos ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrações ==
	


== Experimentos do Elementor ==
	Saída DOM otimizada: Ativo
	Carregamento melhorado de recursos: Ativo
	Carregamento melhorado de CSS: Ativo
	Ícones da fonte inline: Ativo
	Melhorias na acessibilidade: Ativo
	Pontos de quebra adicionais personalizados: Inativo
	admin_menu_rearrangement: Inativo por padrão
	Flexbox Container: Ativo
	Upgrade Swiper Library: Inativo por padrão
	Padrão para Novo Construtor de Tema: Inativo
	Tema Hello - Cabeçalho & Rodapé: Inativo
	Editor V2: Inativo por padrão
	Páginas de Aterrissagem: Inativo
	Elementos aninhados: Inativo por padrão
	Imagens com carregamento gradual: Inativo por padrão
	Salvar como padrão: Ativo por padrão
	Transições da página: Ativo
	Atividades Desempenhadas: Inativo
	Repetir: Ativo
	Envios do formulário: Ativo
	Pressão da Rolagem: Ativo por padrão
	Mega Menu: Inativo por padrão


== Registo ==
	
PHP: showing 7 of 7PHP: 2022-11-25 03:23:13 [notice X 6][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-05 18:43:53 [notice X 42][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] O argumento $finder_category_name está obsoleto desde a versão 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-05 18:57:53 [notice X 126][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] O argumento $control_id está obsoleto desde a versão 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-05 19:08:12 [notice X 5][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] O argumento $control_id está obsoleto desde a versão 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /home/marks.art.br/wip2/251122b/wp-content/plugins/jet-popup/includes/integration.php(134): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2023-01-06 09:49:09 [error X 1][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor-pro/modules/popup/module.php::120] Uncaught Error: Call to a member function save_display_settings_data() on bool in /home/marks.art.br/wip2/251122b/wp-content/plugins/elementor-pro/modules/popup/module.php:120
Stack trace:
#0 [internal function]: ElementorPro\Modules\Popup\Module->save_display_settings()
#1 /home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#2 /home/marks.art.br/wip2/251122b/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#3 /home/marks.art.br/wip2/251122b/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#4 /home/marks.art.br/wip2/251122b/wp-includes/plugin.php(517): WP_Hook->do_action()
#5 /home/marks.art.br/wip2/251122b/wp-admin/admin-ajax.php(188): do_action()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-10 11:51:36 [notice X 3][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor/app/modules/import-export/module.php::563] Trying to get property 'label' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-13 17:04:26 [notice X 2][/home/marks.art.br/wip2/251122b/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::270] Undefined index: post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 9 of 9JS: 2022-12-15 18:19:10 [error X 8][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:694254] T.getContainer is not a function 
JS: 2023-01-06 01:28:31 [error X 1][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view4862\") has already been destroyed and cannot be used. 
JS: 2023-01-07 20:35:36 [error X 10][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.9.2:3:775755] Cannot read properties of undefined (reading \'localized\') 
JS: 2023-01-30 19:43:27 [error X 1][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.9.2:2:14029] An invalid argument supplied as multiple control value [array (
  'category' => 'InvalidElementData',
  'tag' => 'Element `undefined` got as `custom_thumb` value. Expected array or object.',
)]
JS: 2023-02-17 00:23:51 [error X 1][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.9.2:3:518924] this.storage.map is not a function 
JS: 2023-02-18 17:00:57 [error X 4][https://wip2.marks.art.br/251122b/wp-content/plugins/elementor-pro/assets/lib/sticky/jquery.sticky.min.js?ver=3.11.1:1:1162] Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'. 
JS: 2023-02-18 18:29:43 [error X 2][https://donestudiodigital.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1016") has already been destroyed and cannot be used. 
JS: 2023-02-18 18:52:47 [error X 1][https://donestudiodigital.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view5780") has already been destroyed and cannot be used. 
JS: 2023-02-18 20:29:12 [error X 2][https://donestudiodigital.com/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.11.1:2:4825] Cannot read properties of undefined (reading 'attributes') 

Log: showing 14 of 142023-01-05 18:42:40 [info] elementor::elementor_updater Started 
2023-01-05 18:42:40 [info] Elementor/Upgrades - _on_each_version Start  
2023-01-05 18:42:41 [info] Elementor/Upgrades - _on_each_version Finished 
2023-01-05 18:42:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.2',
)]
2023-01-05 18:42:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.2',
)]
2023-02-18 12:58:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.2',
  'to' => '3.11.1',
)]
2023-02-18 12:58:18 [info] elementor::elementor_updater Started 
2023-02-18 12:58:18 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-18 12:58:19 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-18 12:58:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.2',
  'to' => '3.11.1',
)]
2023-02-18 15:09:07 [info] elementor-pro::elementor_pro_updater Started 
2023-02-18 15:09:07 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-18 15:09:07 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-18 15:09:07 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.11.1',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibilidade não especificada
	JetBlog For Elementor: Compatibilidade não especificada
	JetElements For Elementor: Compatibilidade não especificada

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @ofmarconi,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
The method you're using is not correct, therefore is not supported. You're re-initializing an already-defined instance.

### ✅ What you can do to solve your issue:
Please follow the [Developers Documentation](https://developers.elementor.com/docs/hooks/js/#frontend-actions) to correctly hook into Elementor's instance.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor, please feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
